src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
```

Comment:
The current logic applies only the long-term discount if the number of days is greater than or equal to the long-term threshold. However, if the number of days is greater than or equal to both the long-term and short-term thresholds, both discounts should be applied cumulatively. Consider adjusting the logic to account for this.